docs/api/field_type_api_and_best_practices.md

```diff
@@ -1,69 +1,77 @@
 
 `validateFieldSettings()`
 
-Before the Public API stores settings for the `FieldType` in a `FieldDefinition`, the type is asked to validate the settings (which were provided by the user). As a result, the `FieldType` must return if the given settings comply to the schema defined by `getSettingsSchema()`. `validateValidatorConfiguration()` Analog to `validateFieldSettings()`, this method verifies that the given validator configuration complies to the schema provided by `getValidatorConfigurationSchema()`.
+The type is asked to validate the settings (provided by the user) before the Public API stores those settings for the Field Type in a `FieldDefinition`. As a result, the Field Type must return if the given settings comply to the schema defined by `getSettingsSchema()`. 
+`validateValidatorConfiguration()`
+ 
+ Analog to `validateFieldSettings()`, this method verifies that the given validator configuration complies to the schema provided by `getValidatorConfigurationSchema()`.
+
+It is important to note that the schema definitions of the Field Type can be both of arbitrary and serializable format, it is highly recommended to use a simple hash structure. You should follow the [Best practices](../guide/best_practices.md) in order to create future-proof schemas.
 
-It is important to note that while the schema definitions of the `FieldType` maybe both be of arbitrary, serializable format, it is highly recommended to use a simple hash structure. It is highly recommended to follow the [Best practices](../guide/best_practices.md) in order to create future proof schemas.
+!!! note 
 
-**Note:** Since it is not possible to enforce a schema format, the code using a specific `FieldType` must basically know all `FieldType`s it deals with.
+    Since it is not possible to enforce a schema format, the code using a specific Field Type must basically know all Field Types it deals with.
 
-This will also apply to all user interfaces and the REST API, which therefore must provide extension points to register handling code for custom `FieldType`. These extensions are not defined, yet.
+This will also apply to all user interfaces and the REST API, which therefore must provide extension points to register handling code for custom Field Type. These extensions are not defined, yet.
 
 #### Name of the Field Type
 
-If you implement Nameable as an extra service, and register this Service using the tag `nameable`, the method `{FieldType}Nameable->getFieldName()` will be used to retrieve the name.
+If you implement \eZ\Publish\SPI\FieldType\Nameable as an extra service, and register this Service using the tag `ezpublish.fieldType.nameable`, the method `{FieldType}Nameable->getFieldName()` will be used to retrieve the name.
+
+!!! note 
 
-Otherwise the regular `{FieldType}->getName()` method is used.
+    \eZ\Publish\SPI\FieldType\FieldType::getName method is deprecated Since 6.3/5.4.7. 
```

Comment:
Kernel 6.3 equals to Platform v1.3, so earlier than the first LTS (v1.7) and the earliest version we cover in this doc. I think there's no need to specify the version here, just leave `is deprecated`